shared/src/bad_token/trace_call.rs

```diff
@@ -386,7 +386,7 @@ mod tests {
     #[ignore]
     async fn mainnet_tokens() {
         // shared::tracing::initialize("orderbook::bad_token=debug,shared::transport=debug");
-        let http = create_test_transport("https://dev-openethereum.mainnet.gnosisdev.com/");
```

Comment:
I definitely think this is a good change - regardless of how it is implemented, because I was under the impression that this node url is not really meant to be made public.